package.json

```diff
@@ -55,7 +56,7 @@
     "less-middleware": "1.0.3",
     "moment": "2.22.2",
     "mongoose": "4.13.12",
-    "mongoose-version": "github:flashmedia/mongoose-version",
+    "mongoose-version": "idearium/mongoose-version#4ec7aa5b6bd628a7da8e7ab4ce1888bb74ff8b6b",
```

Comment:
@allanchau, we should actually move this to the `linzjs` organisation?

Reply:
It looks like the build on that commit is failing?

Reply:
@smebberson yeah, I fixed a syntax error `'use strict'` in that commit, but I didn't want to change how the code works. I don't think we can fix the tests without doing that because they currently run on an extremely outdated node version 0.X 😱

Reply:
@allanchau, all builds are failing, even those before your updates. Do we even need to version bump the Mongoose dependency? Does that specifically fix anything?

Reply:
It fixes a warning during startup, since we are using it on mongoose 4 already I though it would be fine.

I couldn't get it to build without the `lib/clone-schema.js` change though. At a minimum it needs that.